src/java/relex/PlainTextServer.java

```diff
@@ -258,8 +263,7 @@ else if (args[i].equals("--verbose") )
 						String fin = StanfordView.printRelations(parse, true, "    ");
 						out.println(fin);
 					}
-					if (verbose)
-						System.out.print(SimpleView.printRelations(parse));
+					logger.debug(SimpleView.printRelations(parse));
```

Comment:
To prevent printRelation() call when level is INFO should be

     logger.debug("{}", SimpleView.printRelations(parse));

Reply:
Yes you are right, because it is called before ```.debug()``` call. But ```SimpleView.printRelations(parse)``` may return string which includes placeholders ```{}``` so adding ```"{}"``` format is good idea anyway.

Reply:
I see you point. I have updated the code to use the first argument of the info() and debug() method as a format string.